### DISCUSSION TITLE
图片上传问题

### DISCUSSION THREAD

**Discussion by DomeenoH** [walinejs/waline#690] [Q&A] [answered]
[默认的图床](http://pic.alexhchu.com)貌似down了，想给服务端换成别的上传api好像比较麻烦……官方的hexo插件貌似没有给相应的配置项，想求一个解决方案

**Comment by lizheming** [comment, marked as answer]
1.5.2 的版本修改了下，默认的图片上传改成了 base64 内嵌到评论内容里了，你可以更新下

**Reply by DomeenoH** [reply]
感谢解答！！
